src/Shared/Navigation/MainStack.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Is this stack still needed ? did a serach and it did not come up. detele if not needed

Reply:
Removed